scripts/install

```diff
@@ -50,6 +50,7 @@ fi
 server_name=$(ynh_app_setting_get --app "$synapse_instance" --key server_name)
 domain=$(ynh_app_setting_get --app "$synapse_instance" --key domain)
 mautrix_version=$(ynh_app_upstream_version)
+username_template="sg_{{.}}"
```

Comment:
```suggestion
username_template="signal_{{.}}"
```
I suggest leaving `signal_` by default for new instances, though I don't have a strong opinion on this.

Reply:
I made this change in c6f3962351de174af910254c6a157211335c731d